<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePrescription</class>
 <widget class="QDialog" name="CreatePrescription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>220</width>
     <height>254</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Enter Patient ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="patientIdEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Choose type of prescription</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="prescriptionTypeComboBox">
        <item>
         <property name="text">
          <string>Repeat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>One-off</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Choose type of medication</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="medTypeComboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Choose frequency</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="frequencyComboBox">
        <item>
         <property name="text">
          <string>Everyday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weekyl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fortnightly</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Choose start date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="startDateEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Choose end date</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDateEdit" name="endDateEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="createPrescriptionButton">
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Create Prescription</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Pictures/Icons/create-black-24dp.svg</normaloff>../../Pictures/Icons/create-black-24dp.svg</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
